.eslintrc.js

```diff
@@ -280,6 +280,7 @@ module.exports = {
     'node-core/no-duplicate-requires': 'error',
   },
   globals: {
+    AbortController: 'readable',
     Atomics: 'readable',
```

Comment:
Bikeshed: any reason you picked globally exporting it rather than putting it in a module ?

Reply:
largely because browsers export it as a global and I couldn't come up with a good enough reason not to also in this case. Yes there is the backwards compat / semver-major concern but for 12x and 10x we can easily move it into `util` or hang it off `event` so at least it's there.